docs/modules/ROOT/pages/partials/con_hermetic_verification.adoc

```diff
@@ -0,0 +1,5 @@
+.Verification
+
+. On your application, navigate to the *Activity* > *Pipeline* runs tab.
+
+. For a pipeline run with *Type* as *Build*, the `prefetch-dependencies` stage displays a green check mark indicating that the build process has successfully prefetched all the dependencies.
```

Comment:
You are linking to this for hermetic and pre-fetch verification but you only indicate how to confirm pre-fetch verification. Should you just include the hermetic verification inline there and only use this for the prefetch verification?

Users will often only use prefetch combined with hermetic, but I believe it is _possible_ to do it without hermetic. Do we want to additionally direct the users to a verification of the successful build (important with the prefetch + hermetic case).

Reply:
@arewm your thoughts on this:
Content for hermetic verification: For a pipeline run with *Type* as *Build*, review the build pipeline tasks and scroll down to view the vulnerabilities scan, which summarizes the results of the `clair-scan`.

Content for pre-fetch verification: For a pipeline run with *Type* as *Build*, the `prefetch-dependencies` stage displays a green check mark indicating that the build process has successfully prefetched all the dependencies.

Reply:
I think what Andrew's intent here is that looking to see if the prefetch-dependencies task completed successfully doesn't tell the full story if the intent of setting up prefetch was to use those dependencies to build the component hermetically. A misconfigured prefetch task could think it got everything, but the build task identifies a dependency that wasn't provided, and fail, causing their build to fail.

Clair is unrelated to this as far as I know, what would looking at vulnerability information help to verify for these procedures?

Reply:
To determine if a hermetic build worked properly, you would look at that build-container stage just like you are proposing to look at the prefetch-dependencies stage.